lec12-datasets.Rmd

```diff
@@ -0,0 +1,189 @@
+    3. Opening up, voicing opinions, potential for conflict (which can be good!)
+    4. Stronger bonds form, cooperative
+    5. Greater focus and energy on completion of project
+    6. Team ends after project is successfully completed
+- These stages can be fast-tracked by discussing norms, duties, and conduct early on
+
+[^group_forming]: See [Principles of Group Dynamics](https://open.lib.umn.edu/principlesmanagement/chapter/13-3-group-dynamics/) 
+
+### Roles and responsibilities
+
+- Leader/Facilitator's duty:
+    - Goal is to keep things running smoothly, focused on the task, and on track for time
+    - Keep everyone on topic and on task; stay aware of the time
+    - (Try to) mediate or resolve any conflicts (there will *always* be some type of conflict; how it's dealt with is what matters)
+    - (Try to) encourage everyone to participate and allow everyone a chance at talking
+- Recorder's duty:
+    - Goal is to write/type down main or important points raised or discussed when team is meeting
+    - Keep notes and files organized and orderly
+- Organizer's duty:
+    - Arrange for next meeting time and location
+    - Send reminders to members a day or two before meeting
+
+### Code of conduct
+
+- Vital to establishing boundaries and expectations of being a team member
+    - How do you want each member to treat each other?
+    - How do you deal with conflict?
+    - What is acceptable and unacceptable behaviour?
+- These are outlined in the code of conduct
+- Mostly its common sense (be kind and respectful)
+    - But its important that you as a team write out what everyone wants and agrees to
+
+## Exercise
+
+### As a group, complete these tasks
+
+<!-- show assigned groups -->
+
+- Get into your assigned groups
+- Introduce each other:
+    - Find out everyone's name and year of study
+    - Find out other things about each other (e.g. any plans for next year, etc)
+- Create a two or one word "team name". We'll use this to create a shared folder for everyone to use.
+- Assign roles to each person (these roles will be rotated in every group setting):
+    - You need: facilitator, recorder, organizer
+    - Discuss how and when roles will be rotated *(record it)*
+- Discuss and brainstorm some codes of conduct you want your team to follow *(record it)*
+- Take a few minutes, think about your own skills and what you feel are most competent in
+    - Then, share the top one or two of those skills *(record those skills)*
+- Discuss how responsibilities of each member will be decided on *(record it)*
+
+## Starting the projects
+
+### Datasets available for use
+
+- [Continuous Plankton Recorder Dataset](https://www.gbif.org/dataset/67c54f85-7910-4cbf-8de4-6f0b136a0e34)
+
+    Data on northern hemisphere plankton species, latitude, longitude, date.
+    Going back to 1946. 935 Mb size, almost 2.75 million rows of data.
+
+- [Insecta of Costa Rica](https://www.gbif.org/dataset/3e9817c1-8302-4955-87e3-a408db0ea379)
```

Comment:
The two gbif data sets are quite big with lots of columns... It might confuse students and be hard for us to contain if they go all over the place. On the other hand it's good practice to be able to find a central question in a data set this big.

Reply:
I agree that the largeness might put them off, but I also think it's realistic that you would be looking at data this big so it's good practice.

Reply:
Worth keeping if they do choose it, it will be interesting to see what they zone in on for research question